File: solutions/c/isogram/1/isogram.c
#include "isogram.h"

bool is_isogram(const char phrase[]){
  if(phrase == NULL) return false;
  if(!*phrase) return true;
  int32_t set = { 0 };
  for(; *phrase ; phrase++){
    if(!isalpha(*phrase)) continue;
    int c = tolower(*phrase) - 'a';
    if(set & (1 << c)){
      return false;
    }
    set |= (1 << c);
  }
  return true;
}

File: solutions/csharp/isogram/1/Isogram.cs
public static class Isogram
{
    public static bool IsIsogram(string word)
    {
        word = word.ToLower();
        var set = new HashSet<char>();
        foreach(var c in word)
            if(char.IsLetter(c))
                if(!set.Add(c)) return false;
        return true;      
    }
}
